cmd/README.md

```diff
@@ -49,6 +49,7 @@ Flags:
       --destDir string               Output files dir (default communication-matrix)
       --format string                Desired format (json,yaml,csv,nft) (default "csv")
       --host-open-ports              Generate communication matrix, host open port matrix, and their difference.
+      --isBM                         Platform is baremetal (optional;auto detect if not set).
```

Comment:
not sure I understand the reason for the flag, if we can "auto detect" why would the user choose to mention it explicitly?
can you write the reason in the commit message as well + its title should be shorter e.g. "add .." without the "if it not set code will detect it"